openwisp_controller/config/tests/test_admin.py

```diff
@@ -479,15 +479,6 @@ def test_device_organization_fk_autocomplete_view(self):
             hidden=[data['org2'].name, data['inactive'].name],
         )
 
-    def test_device_templates_m2m_queryset(self):
```

Comment:
Please mention in description why these tests are removed from here

Reply:
I have added in the commit description and in the comment above the reason for removing these test cases.